src/main/java/appeng/recipes/loader/RecipeResourceCopier.java

```diff
@@ -108,15 +108,18 @@ private void copyTo( File destination, String directory ) throws URISyntaxExcept
 		final String[] listing = this.getResourceListing( this.getClass(), directory );
 		for( String list : listing )
 		{
-			if( list.endsWith( ".recipe" ) || list.endsWith( ".html" ) )
+			if( list.endsWith( identifier ) )
 			{
+				// generate folder before the file is copied so no empty folders will be generated
+				FileUtils.forceMkdir( destination );
+
 				this.copyFile( destination, directory, list );
 			}
 			else if( !list.contains( "." ) )
 			{
 				final File subDirectory = new File( destination, list );
-				FileUtils.forceMkdir( subDirectory );
-				this.copyTo( subDirectory, directory + list + "/" );
+
+				this.copyTo( identifier, subDirectory, directory + list + File.separatorChar );
```

Comment:
Seems like it does not like `File.separatorChar`, changing it to a simple `/` works for me.
As paths in java are platform independent, it will even treat `/` as a correct separator for windows, but it seems to have an issue with `\\` for windows. But only when launching via the vanilla launcher, works perfectly find with eclipse.